UI/window-basic-main.cpp

```diff
@@ -1868,7 +1864,18 @@ void OBSBasic::ReceivedIntroJson(const QString &text)
 	bottomLayout->addStretch();
 
 	QVBoxLayout *topLayout = new QVBoxLayout(&dlg);
+
+#if defined(_WIN32) && defined(WITH_OBS_BROWSER)
+		QCefWidget *cefWidget = create_browser_widget(nullptr, info_url);
+		if (!cefWidget) {
```

Comment:
Style:  This `if` statement does not need curly braces.  Remove them.

I know it was this way in the original code.  The style guide says not to do this, so we should avoid introducing or reintroducing code like this where possible.